.travis.yml

```diff
@@ -0,0 +1,33 @@
+os:
+  - linux
+  - osx
+language: perl
+perl:
+  - "5.20"
+  - "5.18"
+  - "5.16"
+  - "5.14"
```

Comment:
I think we should add 5.8. There are still plenty of systems out there that have it as their default version of perl.